doc/cli/cli_intro.rst

```diff
@@ -0,0 +1,158 @@
+    (0008, 0018) SOP Instance UID                    UI: 1.2.777.777.77.7.7777.7777.20030903150023
+    (0008, 0020) Study Date                          DA: '20030716'
+    ...
+
+Note that prefixing the file specification with ``pydicom::`` will read the file
+from the *pydicom* test data files rather than from the normal file system.
+The following examples will use that so that you can replicate these 
+examples exactly.  In normal use, you would leave the ``pydicom::`` prefix
+off when working with your files.
+
+You can also show just parts of the DICOM file by specifying a data element
+using the usual pydicom keyword notation:
+
+.. code-block:: console
+
+    $ pydicom show pydicom::rtplan.dcm::FractionGroupSequence[0]
+    (300a, 0071) Fraction Group Number               IS: "1"
+    (300a, 0078) Number of Fractions Planned         IS: "30"
+    (300a, 0080) Number of Beams                     IS: "1"
+    (300a, 00a0) Number of Brachy Application Setups IS: "0"
+    (300c, 0004)  Referenced Beam Sequence  1 item(s) ----
+    (300a, 0082) Beam Dose Specification Point       DS: [239.531250000000, 239.531250000000, -751.87000000000]
+    (300a, 0084) Beam Dose                           DS: "1.0275401"
+    (300a, 0086) Beam Meterset                       DS: "116.0036697"
+    (300c, 0006) Referenced Beam Number              IS: "1"
+    ---------
+
+You can see the available subcommands by simply typing ``pydicom`` with no
+arguments, or with ``pydicom help``:
+
+.. code-block:: console
+
+    $ pydicom help
+    Use pydicom help [subcommand] to show help for a subcommand
+    Available subcommands: codify, show
+   
+And, as noted in the block above, you get help for a particular subcommand
+by typing ``pydicom help [subcommand]``.  For example:
+
+.. code-block:: console
+
+    $ pydicom help show
+    usage: pydicom show [-h] [-x] [-t] [-q] filespec
+
+    Display all or part of a DICOM file
+
+    positional arguments:
+    filespec              File specification, in format [pydicom::]filename[::element]. If `pydicom::`
+                            prefix is used, then show the pydicom test file with that name. If `element`
+                            is given, use only that data element within the file. Examples:
+                            path/to/your_file.dcm, your_file.dcm::StudyDate,
+                            pydicom::rtplan.dcm::BeamSequence[0], yourplan.dcm::BeamSequence[0].BeamNumber
+
+    optional arguments:
+    -h, --help            show this help message and exit
+    -x, --exclude-private
+                            Don't show private data elements
+    -t, --top             Only show top level
+    -q, --quiet           Only show basic information
+
+
+Installing the pydicom CLI
+--------------------------
+
+The `pydicom` script should automatically be available after you 
+`pip install pydicom`.  It should not require any updates to the system
```

Comment:
What is the 'pydicom script'?

Reply:
Nothing, just slipped.  Should be 'command' to keep consistent.